pkgs/servers/filtron/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "filtron";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "asciimoo";
+    repo = "filtron";
+    rev = "v${version}";
+    sha256 = "18d3h0i2sfqbc0bjx26jm2n9f37zwp8z9z4wd17sw7nvkfa72a26";
+  };
+
+  vendorSha256 = "05q2g591xl08h387mm6njabvki19yih63dfsafgpc9hyk5ydf2n9";
+
+  doCheck = false;
```

Comment:
No tests? Please add a comment.

Reply:
I have tried to fix the tests with a substituteInPlace but it requires much more work than I anticipated, so I marked upstream tests as obsolete and therefore I turned off the tests.